salt/orch/update-etc-hosts.sls

```diff
@@ -27,17 +30,27 @@ etc_hosts_setup:
   salt.state:
     - tgt: {{ updates_all_target }}
     - tgt_type: compound
-    - queue: True
+    # see https://github.com/saltstack/salt/issues/31562
+    - kwarg:
+        queue: True
```

Comment:
Fixing this so `queue: True` is actually picked up may lead to issues. If we're going to merge this, we need to figure out how to validate this won't increase the likelihood of conflcits with the orch/kubernetes.sls or orch/update.sls

At the very least, we should test this change with a upgrade of 5 or 10 nodes - the upgrade orch hits conflicts with this one much more than bootstrap.